index.bs

```diff
@@ -1887,49 +1768,26 @@ const a = builder.input('a', descA);
 const descB = {type: 'float32', dimensions: [4, -1]};
 const b = builder.input('b', descB);
 const c = builder.matmul(a, b);
-const graph = await builder.build({c});
+const graph = builder.build({'c': c});
 
-async function compute(shapeA, shapeB) {
+function compute(shapeA, shapeB, shapeC) {
```

Comment:
For completeness, I propose we add the `sizeOfShape` util function here:
```
function sizeOfShape(array) {
  return array.reduce(
      (accumulator, currentValue) => accumulator * currentValue);
}
```

Reply:
Good point. I fixed it in the new commit.